packages/components/boxes/TertiaryBox.tsx

```diff
@@ -17,6 +17,8 @@ export const TertiaryBox: React.FC<{
         {
           borderWidth: 1,
           borderColor: neutral33,
+          justifyContent: "center",
```

Comment:
Same here are you sure that add this 2 lines doesn't regress other uses ?

Reply:
No, this change will not bring regression